src/components/digit/Digit.tsx

```diff
@@ -0,0 +1,15 @@
+import Button from "../button/Button";
+import { BUTTON_VARIANTS } from "../button/Button.type";
+
+const MAX_NUM = 9;
+const Digit = () => {
+  return (
+    <div className="digits flex">
+      {new Array(10).fill(0).map((_, index) => (
+        <Button variant={BUTTON_VARIANTS.DIGIT}>{MAX_NUM - index}</Button>
+      ))}
```

Comment:
- 숫자 배열은 컴포넌트 외부에서 상수로 선언해도 무방해보입니다. 
- 오름차순 배열을 내림차순으로 바꾸기 위해 index를 빼신 것으로 보이는데 [Array.prototype.toReversed()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed)나 [Array.prototype.toSorted()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toSorted)로 체이닝하는 것이 보기 편할 것 같습니다.
- (개인적으로는) jsx 마크업 내에서 값을 생성하는 것보다 불러온 배열을 그대로 렌더링하는 것이 직관적이라고 생각합니다.

```suggestion
const DIGIT_LIST = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
return (
  <ul>
    {DIGIT_LIST.map((digit) => (
      <Button variant={BUTTON_VARIANTS.DIGIT}>{digit}</Button>
    ))}
  </ul>
)
```